### PR TITLE
Kafka is a direct dependency

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 thoth-python = "*"
 thoth-storages = "*"
 faust = "*"
+kafka = "*"
```
Comment (goern) [inline Pipfile]: why?

Comment (fridex) [reply Pipfile]: It's imported in messages module:

https://github.com/thoth-station/package-update-job/blob/master/messages/hash_mismatch.py#L29-L30

**Comment by KPostOffice** [comment]
I never actually use Kafka so I removed it in PR #7

**Comment by fridex** [comment]
See Kevin's comment, closing - not relevant.

**Pull request closed without merging**